### PR TITLE
Riak 1.4.10

### PR DESCRIPTION
**Pull request by boosty**
Now that Riak 2.0.0 has been released, I think it makes sense to provide a brew version for the latest 1.4 release.

### CODE REVIEW

File: riak1410.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+require "formula"
```
Comment (DomT4) [inline riak1410.rb]: :scissors: this line

File: riak1410.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+class Riak1410 < Formula
+  homepage "http://basho.com/riak/"
+  url "http://s3.amazonaws.com/downloads.basho.com/riak/1.4/1.4.10/osx/10.8/riak-1.4.10-OSX-x86_64.tar.gz"
```
Comment (DomT4) [inline riak1410.rb]: ` https:// ` please.

File: riak1410.rb [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    end
+    inreplace "#{libexec}/etc/app.config" do |c|
+      c.gsub! './data', datadir
+      c.gsub! './log', logdir
+    end
+    bin.write_exec_script libexec/"bin/riak"
+    bin.write_exec_script libexec/"bin/riak-admin"
+    bin.write_exec_script libexec/"bin/riak-debug"
+    bin.write_exec_script libexec/"bin/search-cmd"
+  end
```
Comment (DomT4) [inline riak1410.rb]: Any chance of a test down here? Thanks!

**Comment by DomT4** [comment]
If this is still useful please address the comments I made above and anything ` brew audit --strict ` highlights on your local machine. Thanks!

**Comment by DomT4** [comment]
Closing per inactivity. Please feel free to request reopening or open a new PR if you're still interested - If you do, please fix any comments ` brew audit --strict ` has, Thanks!

**Comment by boosty** [comment]
Thanks for the review! I did not know about `brew audit`. Will try to change the formula in the next days.

**Comment by DomT4** [comment]
No problem. ` brew audit --strict /path/to/local/formula.rb ` will flag up any audit failures :). Either push to this PR again and give me a nudge when you've done so and I'll reopen, or open a new PR with the changes. Whatever you're comfortable with works for me.

**Pull request closed without merging**